packages/shared/src/components/buttons/ToggleClickbaitShield.tsx

```diff
@@ -19,8 +19,10 @@ import { FeedSettingsMenu } from '../feeds/FeedSettings/types';
 
 export const ToggleClickbaitShield = ({
   origin,
+  buttonProps,
 }: {
   origin: Origin;
+  buttonProps?: ButtonProps<'button'>;
```

Comment:
So the app won't break if we pass null or undefined`.
```suggestion
  buttonProps?: ButtonProps<'button'> = {};
```